.github/workflows/release.yaml

```diff
@@ -99,25 +102,35 @@ jobs:
   windows-depends:
     strategy:
       matrix:
-        os: [windows]
+        os: [windows-2019, windows-2025]
         arch: [amd64]
         preset: ['CPU']
         include:
           - os: windows
             arch: amd64
             preset: 'CUDA 11'
-            install: https://developer.download.nvidia.com/compute/cuda/11.3.1/local_installers/cuda_11.3.1_465.89_win10.exe
-            cuda-version: '11.3'
+            install: https://developer.download.nvidia.com/compute/cuda/11.8.0/local_installers/cuda_11.8.0_522.06_windows.exe
+            cuda-version: '11.8'
           - os: windows
             arch: amd64
             preset: 'CUDA 12'
             install: https://developer.download.nvidia.com/compute/cuda/12.4.0/local_installers/cuda_12.4.0_551.61_windows.exe
             cuda-version: '12.4'
+          - os: windows
+            arch: amd64
+            preset: 'CUDA 12'
+            install: https://developer.download.nvidia.com/compute/cuda/12.6.3/local_installers/cuda_12.6.3_561.17_windows.exe
+            cuda-version: '12.6'
+          - os: windows
+            arch: amd64
+            preset: 'CUDA 12' # Blackwell
+            install: https://developer.download.nvidia.com/compute/cuda/12.8.0/local_installers/cuda_12.8.0_571.96_windows.exe
+            cuda-version: '12.8'
```

Comment:
Adding additional branches to the build matrix isn't necessary. When we update Ollama for Blackwell, bumping CUDA version from 12.4.0 to 12.8.0 should be sufficient.

As implemented here, the build will fail since these two new jobs will conflict with the existing CUDA 12 job in build artifacts.